File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_3/InsertionSort.java
import java.util.* ;
import java.io.*;

public class InsertionSort {

    public static void insertionSort(int n , int[] arr) {
        // Write your code here.
        for(int i=1; i<n;i++){
            int j = i-1;
            int key = arr[i];
            while(j>=0&&arr[j]>key){
                arr[j+1]= arr[j];
                j=j-1;
            }
            arr[j+1]=key;
        }

    }
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_3/Insertion_Sort.py
from os import *
from sys import *
from collections import *
from math import *

from typing import List

def insertionSort(n: int, arr: List[int]) -> None:
    # Write your code here.
    for i in range(1, len(arr)):
        k = arr[i]
        j = i - 1

        while j >= 0 and k < arr[j]:
            arr[j + 1] = arr[j]
            j -= 1

        arr[j + 1] = k
    return arr
    pass